src/AI/agents/agents/services/telemetry/json_utils.py

```
@@ -0,0 +1,13 @@
+"""Simple JSON utility functions."""
+import json
+
+
+def is_json(text):
+    """Check if a string is valid JSON"""
+    if not isinstance(text, str):
+        return False
+    try:
+        json.loads(text)
+        return True
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)